coins.json

```diff
@@ -501,5 +501,16 @@
         "publicKeyType": "secp256k1",
         "transactionPath": "/tx/",
         "explorer": "https://steemblockexplorer.com"
+    },
+    {
+        "id": "eosio",
+        "name": "EOSIO",
+        "symbol": "EOS",
+        "decimals": 18,
```

Comment:
should be 3, same as Steem

Reply:
sorry, seems be 4

Reply:
Agreed. Changing it to 4. However, this code supports all tokens listed on EOS blockchain which can have up to 18 decimals.